src/proxy.conf.json

```diff
@@ -0,0 +1,5 @@
+{
+  "/api/v1": {
```

Comment:
we should align on one way how BE calls will be made - it's done differently in FeedbackService (`backendUrl` is used instead of proxy)

Reply:
I tried using environment files but I ended up getting an error for every request.

Reply:
@Burhak I personally went with ugly solution on backend:
`@CrossOrigin(origins = "http://localhost:4200")` anotation for controller/controller method i was using at the moment (of course without commiting it). 

There are also different possible backend solutions which you can apply - i think implementing separate configuration bean with webmvcconfigurer and something like `@Profile("dev")`

Since issue you have is only relevant for local development afaik, i would not include your solution in angular github repository even if it is also valid.